libs/etterlatte-regler/src/main/kotlin/regler/RegelDsl.kt

```diff
@@ -91,7 +105,7 @@ fun <G, S> definerKonstant(
     verdi
 )
 
-infix fun <G, S> RegelMeta.kombinerer(regel1: Regel<G, S>) = this to regel1
+infix fun <G, S> RegelMeta.benytter(regel1: Regel<G, S>) = this to regel1
```

Comment:
Litt usikker på denne endringen. 

Bakgrunnen var at jeg ville slippe å ha to funksjoner som gjør akkurat det samme, men da måtte jeg endre navn på den til å dekke bruken med bare en annen regel også. 

Andre forslag til navn som kan være bedre?

Reply:
Synes det er riktig å tenke i denne retningen. Er jo egentlig en mapping over resultatet av 1, 2 og 3 regler vi støtter her. Ingen grunn til at de skal ha forskjellig navn. Sånn sett så burde kanskje også reglene som defineres også hete noe likt?